dept44-starter/src/main/resources/config/application.properties

```diff
@@ -22,6 +22,7 @@ logging.file.path=log
 logging.level.se.sundsvall.dept44.payload=TRACE
 logbook.default.logger.name=se.sundsvall.dept44.payload
 logbook.default.excluded.paths=/,**/webjars/**,**${springdoc.api-docs.path}**,**/swagger-resources,**/swagger-resources/**,**/error,**/csrf,**/swagger-ui.html,**/swagger-ui/**,**/favicon.ico,**/actuator,**/actuator/**,**/h2-console/**
+#logbook.secure-filter.enabled=false
```

Comment:
Is this property needed or not?

Reply:
Nope. A leftover from some bug-hunting that shouldn't be there